lib/cfg-grammar.y

```diff
@@ -1103,6 +1107,10 @@ source_reader_option_flags
 	|
 	;
 
+driver_option
+    : KW_PERSIST_ID '(' string ')' { log_pipe_set_persist_id(&last_driver->super, g_strdup($3)); free($3); }
+    ;
+
 threaded_dest_driver_option
```

Comment:
The indentation character is already mixed, I would clarify this in a new pull request.